tool_collections/samtools/samtools_stats/samtools_stats_options.py

```diff
@@ -0,0 +1,44 @@
+import pysam
+import os
+import tempfile
```

Comment:
Hopefully can actually get rid of these code files and use the newly extended BAM metadata :)